web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -245,10 +253,27 @@ export function ClusterResources({
     clear();
   }
 
+  const { setInfoGuideConfig } = useInfoGuide();
+  function onShowStatusInfo(resource: UnifiedResourceDefinition) {
+    openStatusInfoPanel({
+      isEnterprise: cfg.edition === 'ent',
```

Comment:
what will `isEnterprise` affect?

Reply:
it sets this [field](https://github.com/gravitational/teleport/pull/54475/files#diff-5e92f33068cb57a641154d6f26a85acc0b058f62af74977f6cf4b497c311812eR40) to true. the comment explains the reason for that flag i linked, but the side panel acts like a  global modal inside a root container (for web UI it's the Main.tsx), where when rendered, the root container adds space to push contents aside for the panel, but for unified resources enterprise version, a side panel already exists for the request checkout, so we don't need the root container to push aside some space